### DISCUSSION TITLE
Error to send Campains in Gophish

### DISCUSSION THREAD

**Discussion by NoemiVilla** [gophish/gophish#3093] [Q&A]
Hello Guys 

A have a question, I've been working with Gophish to make a project, at first it worked great, but recently in testing a changed the IP "listen_url": "127.0.0.1:3333", for an external IP, but when I send the campaign it fails and the mails aren't delivered, and in the dashboard show "Error". 
Does anyone have the same error? or something like this?

**Comment by newtothisfornow** [comment]
im new to phishing and couldnt find a community so i made one on discord add me at usertenthousand or join the server here https://discord.gg/REujkWET

**Reply by Zzulfaqar** [reply]
Hi can i get the new link thanks

**Reply by noah369-ux** [reply]
Hello, it seems the Discord link is no longer functioning. Is this group still operational, or has it ventured into the digital ether?
